gp-includes/assets-loader.php

```diff
@@ -60,7 +60,7 @@ function gp_enqueue_styles( $handles ) {
  * @param string $handle A single style handle to enqueue.
  */
 function gp_enqueue_style( $handle ) {
-	if ( ! in_array( $handle, GP::$styles ) ) {
+	if ( false === in_array( $handle, GP::$styles ) ) {
```

Comment:
OK, that was for the third argument of `in_array()`. `in_array()` returns only boolean values so there is no need for the strict check.